cli/tools/init/mod.rs

```diff
@@ -203,7 +211,7 @@ Deno.test(function addTest() {
     info!("  deno task dev");
     info!("");
     info!("  {}", colors::gray("# Run the tests"));
-    info!("  deno -R test");
+    info!("  deno test -R");
```

Comment:
This is wrong, the allow read permission will not be applied in such a command

Reply:
I'm not sure if I'm misunderstanding your comment, please let me know if I am

I will test some more, but I have deno 1.46 along with DENO_FUTURE=1 and this was the correct order of args.

I think the integration test would have failed also if this was wrong.

Reply:
I was wrong about the integration tests, it appears the read permission flag doesn't have any effect on the child process running in these tests and read permission implicitly granted.

Testing manually again though `deno test -R` is needed for read permission

```
deno 1.46.2 (stable, release, aarch64-apple-darwin)
v8 12.9.202.5-rusty
typescript 5.5.2
```

```bash
deno init --serve demo
cd demo
```

Fix route handler

```diff
{
  pattern: new URLPattern({ pathname: "/static/*" }),
-   handler: (req) => serveDir(req, { urlRoot: "./" }),
+   handler: (req) => serveDir(req),
// or -- handler: (req) => serveDir(req, { fsRoot: "static", urlRoot: "static" }),
},
```

`deno -R test`

```
cotyhamilton dev/demo 
➜ deno -R test
Check file:///Users/cotyhamilton/dev/demo/main_test.ts
running 4 tests from ./main_test.ts
serverFetch ... ok (1ms)
serverFetchNotFound ... ok (0ms)
serverFetchUsers ... ok (0ms)
serverFetchStatic ...
------- output -------
Requires read access to "static/main.ts", run again with the --allow-read flag
[2024-09-01 03:54:26] [GET] /static/main.ts 500
----- output end -----
serverFetchStatic ... ok (6ms)

ok | 4 passed | 0 failed (10ms)
```

> Requires read access to "static/main.ts", run again with the --allow-read flag

---

`deno test -R`

```
cotyhamilton dev/demo 
➜ deno test -R
running 4 tests from ./main_test.ts
serverFetch ... ok (1ms)
serverFetchNotFound ... ok (0ms)
serverFetchUsers ... ok (0ms)
serverFetchStatic ...
------- output -------
No such file or directory (os error 2): stat 'static/main.ts'
[2024-09-01 03:54:32] [GET] /static/main.ts 404
----- output end -----
serverFetchStatic ... ok (2ms)
```

Reply:
@bartlomieju, `deno test -R` is actually right. I think you're getting confused because there's no defined entry point. Imagine it was `deno test -R <entrypoint>`. [Reference](https://docs.deno.com/runtime/manual/tools/test/).